contracts/consumer/converter/src/contract.rs

```diff
@@ -30,8 +30,8 @@ pub struct ConverterContract<'a> {
 
 #[cfg_attr(not(feature = "library"), sylvia::entry_points)]
 #[contract]
-#[error(ContractError)]
-#[messages(converter_api as ConverterApi)]
+#[sv::error(ContractError)]
+#[sv::messages(converter_api as ConverterApi)]
```

Comment:
Since `0.10.0` below is possible since we can deduce the interface name by converting the path to camel case

```suggestion
#[sv::messages(converter_api)]
```